elements/pfe-toast/src/pfe-toast.scss

```diff
@@ -0,0 +1,109 @@
+@import "../../pfe-sass/pfe-sass";
+
+:host {
+  // local variables
+  --pfe-toast--Top: 50px;
+  --pfe-toast--MaxWidth: 500px;
+  --pfe-toast--MinWidth: 250px;
+  --pfe-toast--Right: 50px;
+  --pfe-toast--Right--offset: 50px; // needed with box-shadow
+  --pfe-toast__container--BackgroundColor: #fff;
+  --pfe-toast__container--BoxShadow: rgba(3, 3, 3, 0.13) 0px 3px 7px 3px, rgba(3, 3, 3, 0.12) 0px 11px 24px 16px;
+  --pfe-toast__container--Color: #151515;
+  --pfe-toast__container--Padding: 1rem;
+  --pfe-toast__close--svg--Fill: #151515;
+  --pfe-toast__close--svg--hover--Fill: #151515;
+  --pfe-toast__close--svg--Height: 12px;
+  --pfe-toast__close--svg--Width: 12px;
+  --pfe-toast__close--PaddingTop: 0;
+  --pfe-toast__close--PaddingRight: 0;
+  --pfe-toast__close--PaddingBottom: 0;
+  --pfe-toast__close--PaddingLeft: 0;
+
+  // flex parent props
+  align-items: stretch;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: flex-start;
+  
+  // animation
+  position: absolute;
+  right: calc(-1 * (var(--pfe-toast--MaxWidth) + var(--pfe-toast--Right--offset)));
+  top: var(--pfe-toast--Top);
+  transition: right 0.5s ease-in-out;
+  will-change: right;
+  
+  max-width: var(--pfe-toast--MaxWidth); // media queries?
+  min-width: var(--pfe-toast--MinWidth);
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host(.open) {
+  right: var(--pfe-toast--Right);
```

Comment:
Try to avoid redeclaring properties once they've been set on host and instead, updating the variables that they use when state or context changes.  You can see this in pfe-cta starting on line 181 as an example.

Reply:
Sure, but why is that?

Reply:
By declaring the CSS property once and updating the variable assigned to it throughout the rest of the file, it makes it easier for us to see/update where that property is used later on. If background color is applied to the host for example and later we want to move that to a wrapper class in the template, we only have to move that property once and we know the updates will be retained. Also, because they cascade so nicely, you can update a regional variable (like --pfe-toast__close) on the host for state changes like `:host(:hover)` without having to target the class for the close element again. Let's cover this more when we all meet to chat about Sass!